roles/edpm_update/tasks/containers.yml

```diff
@@ -0,0 +1,105 @@
+---
+
+- name: Login to container registries if needed
+  ansible.builtin.include_role:
+    name: osp.edpm.edpm_podman
+    tasks_from: login.yml
+  tags:
+    - edpm_podman
+    - edpm_update
+
+- name: Updates containers for edpm_iscsid role
+  ansible.builtin.include_role:
+    name: osp.edpm.edpm_iscsid
+    tasks_from: run.yml
+  tags:
+    - edpm_iscsid
+    - edpm_update
+  when: '"nova-storage" in edpm_services'
+
+- name: Updates containers for edpm_ovn role
+  ansible.builtin.include_role:
+    name: osp.edpm.edpm_ovn
+    tasks_from: run.yml
+  tags:
+    - edpm_ovn
+    - edpm_update
+  when: '"ovn" in edpm_services'
```

Comment:
Should we handle override like we are doing in download_cache? (see last update as for @fao89 suggestion)

https://github.com/openstack-k8s-operators/edpm-ansible/pull/637/files#diff-24c4ac4687e3ee39c2855757e6a8ed75ca0a2b62aec0dc242d8e8322065ae6a8

Reply:
Yes, I'd add the same handling here